docs/configure/ci/autoscan.md

```diff
@@ -20,10 +57,35 @@ To re-run an Auto-Scan:
 
 Re-running an Auto-Scan job usually takes a few minutes. Look for the 'Completed' status to see your latest results.
 
-## Customizing Auto-Scan Using Fairwinds-insights.yaml
-Unlike the Insights CI integration, Auto-Scan does not require users to create a `fairwinds-insights.yaml` configuration file at the base of their GitHub repository. This is because Auto-Scan will automatically crawl and discover YAML manifests, Helm charts, and docker images available for scanning.
+## Manifests auto-discovery
+Unlike Insights CI integration, Auto-Scan does not require users to create a `fairwinds-insights.yaml` configuration file at the base of their GitHub repository. This is because Auto-Scan will automatically crawl and discover YAML manifests, Helm charts, and docker images available for scanning.
+
+### Reports Configuration
+When using auto-discovery, all reports are `enabled` by default, if you want to further customize. It is possible to manually configure it via API using CURL:
```

Comment:
When using auto-discovery, all reports are `enabled` by default. To customize this, you can manually configure via API using CURL: